rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -126,7 +132,7 @@ def __init__(
         self._load_params()
 
         # transform numbers to labels
-        self.inv_label_dict = inv_label_dict
+        self.inverted_label_dict = inv_label_dict
```

Comment:
```suggestion
        self.inverted_label_dict = inverted_label_dict
```
and in init as well then